### PR TITLE
pkg/kubelet: kill a TODO in util.go

### PR DESCRIPTION
**Pull request by xiang90**
Move kubelet.CapabilitiesSetup to capabilities.Setup

### CODE REVIEW

File: pkg/capabilities/capabilities.go [hunk 1]
```diff
@@ -40,6 +40,14 @@ func Initialize(c Capabilities) {
 	})
 }
 
+// Setup the capability set.  It wraps Initialize for improving usibility.
+func Setup(allowPrivileged bool, hostNetworkSources []string) {
```
Comment (thockin) [inline pkg/capabilities/capabilities.go]: I'm not a fan of this factoring, but it's not your problem.  @erictune @vmarmol things like `hostNetworkSources` is pretty kubelet specific, why is this outside pkg/kubelet?

**Comment by thockin** [comment]
LGTM

File: pkg/capabilities/capabilities.go (hunk 1 above)
Comment (vmarmol) [inline pkg/capabilities/capabilities.go]: We didn't have a better concept at the time and the standard was evolving. With the global capabilities object this doesn't make much sense outside of the Kubelet.

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**